tasks/Onlooker.tex

```diff
@@ -0,0 +1,57 @@
+\section{Onlooker [Housekeeper]}
+Robot observes activities performed by an individual and reports them. 
+
+\subsection{Main Goal}
+Robot reports the sequence of activities performed by a person.
+
+\noindent\textbf{Reward:} 500pts (100 pts per activity).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item 500 pts for reporting the sequence of activities for the second person (100 pts per activity)
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The test takes place in the living room.
+
+	\item \textbf{Start Location}: The robot starts at a predefined location.
+
+	\item \textbf{Persons}: There are two people. At the start, the main person to look for is standing, and the second person is sitting.
+\end{itemize}
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Partial scoring:} The task allows partial scoring for each reported activity.
+	
+	\item \textbf{Activity List:} talk to someone, talk on a phone call, reading, writing, relaxing, walking, drinking, and using laptop.
```

Comment:
- ✔️ Phone call 
- ✔️ Reading
- ⚠️ Writing: i.e. taking notes with pen and paper?
- ⚠️ Relaxing: Ambiguous, since reading a book can be relaxing, as well as listening to music, etc. I'd change it to watching TV
- ⚠️ Walking: Might seem forced without a purpose. Some people walk while on the phone.
- ⚠️ Drinking: As in actually drinking from a bottle or as in holding a glass with vodka?
- ✔️ Using laptop
- ❌ Talk to someone: Can't do if there are only two people in the room and both are in mutually exclusive activities

Some other activities to include:
- Serving drinks
- Sleeping

Reply:
Yes, writing will mean taking notes with pen and paper. May be writing couple of sentences would be enough.

I agree, relaxing can be ambiguous. But as you suggested we can replace it with sleeping which would be a more clear form of relaxing. Also, it would help if we restrict sleeping to sleeping (lying down).

I agree walking may seem purposeless and can be removed from the list of activities. However, we should somehow force the people to move around. Otherwise, it would not be interesting if a person does five activities while sitting or standing on the same location. There can be a rule saying people will move around the house to do different activities. And the referee can distribute the activities around the living room. For example, the person will read and sleep on sofa, write on the table, use laptop at the desk, drink from a glass of water placed on the kitchen counter, etc.

Drinking would be more interesting if someone actually drink from a glass or bottle. Otherwise, it could be the case that the person is holding the drink for someone else.

The reason to include 'talk to someone' was to recognize a group activity. This can be the only mutual activity between the two people. And before and after 'talk to someone' each person can do individual activities. I think it will be challenging specially when two people are doing activities in the same room. 

Serving drinks would be a mutual activity. But it would not be the same for both people because one would be serving and one would be receiving. Some other examples of mutual (group) activities could be playing board game and moving table together.

Reply:
exercise (push ups or squats) can an interesting activity to report.

Reply:
I mostly like some of your ideas, but I must admit that although they sound appealing for a lab setup of 2~3 runs, they are completely unfeasible in the competition.

I can ask  volunteer to sleep for the 3 hours a block last, but exercising, running from here to there, and other kind of physically demanding activities, I don't think so. Same for drinking. I can hold a glass of water for 3 hours, but I can't drink more than a couple of sips every 30 seconds for more than half an hour before needing urgently to make a technical step-over.

Reply:
You are right those activities would be hard to repeat. Besides, recognizing that a person is holding a glass or bottle would be enough to ascertain that the person is drinking because there would not be any other activity which would involve holding a glass. Therefore, there is no need of actual drinking.

So the list of activities will be,

Phone call
Reading
Writing
Sleeping
Drinking
Using laptop

We should have at least two more activities (i.e., eight in total) so that random guessing of activities would be hard. However, it will be good to have more activities.

How about 'play with a ball' and 'clean a table'?